content/_index.md

```diff
@@ -1,8 +1,10 @@
 ---
 hero:
   title: The Free Software Media System
-  text: Jellyfin is the Free Software Media System that puts you in control of managing and streaming your media. There are no strings attached, no premium licenses or features, and no hidden agendas.
```

Comment:
I'd prefer to keep this wording. Mentioning our FOSS-ness is really important, moreso than the following sentence.

Reply:
That term is right above the description though, in a much larger font, so I thought it seemed repetitive.

Reply:
Oh right I forgot about that subtitle. All good.